sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/README.md

```diff
@@ -1,104 +1,146 @@
-# README.md template
+# Azure Schema Registry Apache Avro library for .NET
 
-Use the guidelines in each section of this template to ensure consistency and readability of your README. The README resides in your package's GitHub repository at the root of its directory within the repo. It's also used as the package distribution page (NuGet, PyPi, npm, etc.) and as a Quickstart on docs.microsoft.com. See [README-EXAMPLE.md](README-EXAMPLE.md) for an example following this template.
+This library provides an Apache Avro serialization and deserialization API using the Azure Schema Registry service.
 
-**Title**: The H1 of your README should be in the format: `# [Product Name] client library for [Language]`
+## Getting started
 
-* All headings, including the H1, should use **sentence-style capitalization**. Refer to the [Microsoft Style Guide][style-guide-msft] and [Microsoft Cloud Style Guide][style-guide-cloud] for more information.
-* Example: `# Azure Batch client library for Python`
+### Install the package
 
-# Azure Template client library for .NET
+Install the Azure Schema Registry Apache Avro library for .NET with [NuGet][nuget]:
 
-**Introduction**: The introduction appears directly under the title (H1) of your README.
+```PowerShell
+Install-Package Microsoft.Azure.Data.SchemaRegistry.ApacheAvro
+```
 
-* **DO NOT** use an "Introduction" or "Overview" heading (H2) for this section.
-* First sentence: **Describe the service** briefly. You can usually use the first line of the service's docs landing page for this (Example: [Cosmos DB docs landing page](https://docs.microsoft.com/azure/cosmos-db/)).
-* Next, add a **bulleted list** of the **most common tasks** supported by the package or library, prefaced with "Use the client library for [Product Name] to:". Then, provide code snippets for these tasks in the [Examples](#examples) section later in the document. Keep the task list short but include those tasks most developers need to perform with your package.
-* Include this single line of links targeting your product's content at the bottom of the introduction, making any adjustments as necessary (for example, NuGet instead of PyPi):
+### Prerequisites
 
-  [Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-batch) | [Package (PyPi)](https://pypi.org/project/azure-batch/) | [API reference documentation](https://docs.microsoft.com/python/api/overview/azure/batch?view=azure-python) | [Product documentation](https://docs.microsoft.com/azure/batch/)
+* An [Azure subscription][azure_sub]
```

Comment:
This link isn't defined.

Reply:
Added the reference link. Also, you use square brackets for reference links and parens for in-place links.